cmd/restore-agent/main.go

```diff
@@ -157,7 +158,7 @@ func main() {
 		SkipHashCheck:       false,
 	}
 
-	client := snapshot.NewV3(nil)
+	client := snapshot.NewV3(zapLogger)
```

Comment:
The etcd client logs were coming out in a different format to the restore-agent. I've made it so that they both use the same underlying zap logger:

```
kubectl -n teste2e-parallel-restore logs restored-cluster-0-restore 
2020-02-26T16:53:31.681Z	INFO	setup	Starting restore-agent	{"version": "v0.2.0-22-g381131f", "etcd-peer-name": "restored-cluster-0", "etcd-cluster-name": "restored-cluster", "etcd-initial-cluster": "restored-cluster-0=http://restored-cluster-0.restored-cluster.teste2e-parallel-restore.svc:2380,restored-cluster-1=http://restored-cluster-1.restored-cluster.teste2e-parallel-restore.svc:2380,restored-cluster-2=http://restored-cluster-2.restored-cluster.teste2e-parallel-restore.svc:2380", "etcd-advertise-url": "http://restored-cluster-0.restored-cluster.teste2e-parallel-restore.svc:2380", "etcd-data-directory": "/var/lib/etcd", "snapshot-directory": "/tmp/snapshot", "proxy-url": "eco-proxy.eco-system.svc:80", "backup-url": "s3://backups.test.improbable.io/snapshot.db?endpoint=http://minio.minio.svc:9000&disableSSL=true&s3ForcePathStyle=true&region=eu-west-2", "timeout": 300}
2020-02-26T16:53:31.681Z	INFO	restore-agent	Dialing proxy
{"level":"info","msg":"resolved URL Host","url":"http://restored-cluster-0.restored-cluster.teste2e-parallel-restore.svc:2380","host":"restored-cluster-0.restored-cluster.teste2e-parallel-restore.svc:2380","resolved-addr":"127.0.0.1:2380"}
{"level":"info","msg":"resolved URL Host","url":"http://restored-cluster-0.restored-cluster.teste2e-parallel-restore.svc:2380","host":"restored-cluster-0.restored-cluster.teste2e-parallel-restore.svc:2380","resolved-addr":"127.0.0.1:2380"}
{"level":"info","msg":"restoring snapshot","path":"/tmp/snapshot/snapshot.db","wal-dir":"/tmp/snapshot/data-dir/member/wal","data-dir":"/tmp/snapshot/data-dir","snap-dir":"/tmp/snapshot/data-dir/member/snap"}
{"level":"info","msg":"added member","cluster-id":"11d332b024f04651","local-member-id":"0","added-peer-id":"5867d241afdc832","added-peer-peer-urls":["http://restored-cluster-2.restored-cluster.teste2e-parallel-restore.svc:2380"]}
{"level":"info","msg":"added member","cluster-id":"11d332b024f04651","local-member-id":"0","added-peer-id":"8915fe400f35fec3","added-peer-peer-urls":["http://restored-cluster-1.restored-cluster.teste2e-parallel-restore.svc:2380"]}
{"level":"info","msg":"added member","cluster-id":"11d332b024f04651","local-member-id":"0","added-peer-id":"bc92bd5746ac7bee","added-peer-peer-urls":["http://restored-cluster-0.restored-cluster.teste2e-parallel-restore.svc:2380"]}
{"level":"info","msg":"restored snapshot","path":"/tmp/snapshot/snapshot.db","wal-dir":"/tmp/snapshot/data-dir/member/wal","data-dir":"/tmp/snapshot/data-dir","snap-dir":"/tmp/snapshot/data-dir/member/snap"}

```